app/views/feeds/_form_image.html.erb

```diff
@@ -0,0 +1,29 @@
+<%= turbo_frame_tag "itunes-images", data: {controller: "morphdom"} do %>
```

Comment:
hmm, i copied these turbo frames over from `episode/_form_image` but i'm not sure what they are for? i tried with testing with them commented out and i couldn't discern the difference.

Reply:
I think if you remove these, there's no turbo-frame wrapping the individual images at any level.

So when you click the delete button (or upload a new image) ... the entire page navigates, and you end up with a bunch of query params in your browser url.

With the turbo frame in there, just that frame navigates/refreshes.